models/balancer/arbitrum/balancer_v2_arbitrum_trades.sql

```diff
@@ -0,0 +1,228 @@
+{{ config(
+    schema = 'balancer_v2_arbitrum',
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook = '{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "balancer_v2",
+                                \'["mendesfabio", "jacektrocinski", "thetroyharris"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2021-08-27' %}
+
+WITH    
+    fees_base AS (SELECT *, MAX(index) OVER(PARTITION BY tx_hash, contract_address) AS max_index_same_tx FROM balancer_v2_arbitrum.pools_fees),
```

Comment:
please use jinja syntax on all sources / references throughout. another example is line 27
@thetroyharris

Reply:
On it. Apologies for that as it was a late addition. Will make sure to better check for the jinja syntax moving forward.

Reply:
Done across all chains, thank you.